cpp/src/arrow/io/io-file-test.cc

```diff
@@ -41,14 +41,34 @@ static bool FileExists(const std::string& path) {
   return std::ifstream(path.c_str()).good();
 }
 
+#if defined(_MSC_VER)
+void customInvalidParameterHandler(const wchar_t* expression,
+	const wchar_t* function,
+	const wchar_t* file,
+	unsigned int line,
+	uintptr_t pReserved)
```

Comment:
We don't use Hungarian notation (https://google.github.io/styleguide/cppguide.html#Windows_Code), see note below though